src/wireclient.js

```diff
@@ -93,10 +93,19 @@
   }
 
   function readBinaryData(content, mimeType, callback) {
-    var blob = new Blob([content], { type: mimeType });
+    var blob;
+    global.BlobBuilder = global.BlobBuilder || global.WebKitBlobBuilder;
```

Comment:
Good point. Albeit if it re-assigns it to the prefixed one, there's no harm really, because it's the same thing.